build.gradle

```diff
@@ -87,9 +78,18 @@ allprojects {
     apply from: "$rootDir/gradle/rat.gradle"
 
     repositories {
+        mavenLocal()
         jcenter()
         mavenCentral()
-        mavenLocal()
+        maven {
+            url "https://oss.jfrog.org/jfrog-dependencies"
+        }
+        maven {
+            url "https://repository.apache.org/snapshots"
+        }
+        maven {
+            url "https://plugins.gradle.org/m2/"
```

Comment:
I did not understand the need for plugins repo here ?

Reply:
removing it